examples/config-dev.yaml

```diff
@@ -50,6 +50,14 @@ web:
   # tlsCert: /etc/dex/tls.crt
   # tlsKey: /etc/dex/tls.key
 
+# Configuration for dex appearance
+# frontend:
+  # issuer: Dex
+  # issuerUrl: http://localhost:5556
+  # logoURL: theme/logo.png
```

Comment:
Are `Url` and `URL` case sensitive fields? I think we should be more consistent about naming config parameters. If it's case sensitive and the fields are named differently...that's life, otherwise let's use the same casing (honestly, don't care which one)

Reply:
They are both case-insensitive, I think.

Reply:
Oh, I just accidentally copied the issuerUrl option from somewhere. It turns out that it doesn't exist.